driver-core/src/test/functional/com/mongodb/operation/CreateIndexesOperationSpecification.groovy

```diff
@@ -72,6 +72,20 @@ class CreateIndexesOperationSpecification extends OperationFunctionalSpecificati
         async << [true, false]
     }
 
+    def 'should support maxTime'() {
```

Comment:
Can this use ClusterFixture#enableMaxTimeFailPoint like other similar maxTimeMS integration tests?

Reply:
Done